### DISCUSSION TITLE
Mod Support request: Cosmetic Armors

### DISCUSSION THREAD

**Discussion by Luuc4sFlp** [xnebulr/Minecraft-Default-Dark-Mode#215] [Ideas]
That's pretty much it, just a request of mod support for [Cosmetic Armor](https://modrinth.com/mod/cosmetic-armor).

**Comment by aricloverEXTRA** [comment]
Hey @Luuc4sFlp Use https://modrinth.com/resourcepack/default-dark-mode-expansion along with NebuIr’s **Default Dark Mode**.

**Cosmetic Armor** support is included in **Default Dark Mode: Expansion** for now and hope it works fine! ✨
